src/frontend/js/components/PaymentInterfaces/Dummy.tsx

```diff
@@ -0,0 +1,43 @@
+import { handle } from 'utils/errors/handle';
+import { useAsyncEffect } from 'hooks/useAsyncEffect';
+import { PaymentErrorMessageId, PaymentInterfaceProps } from 'components/PaymentInterfaces/types';
+
+/**
+ * !!! DEVELOPMENT PURPOSE ONLY !!!
+ *
+ * Load the dummy payment interface
+ *
+ */
+
+const DummyPayment = ({ url, payment_id, onSuccess, onError }: PaymentInterfaceProps) => {
+  useAsyncEffect(async () => {
+    try {
+      const response = await fetch(url, {
+        body: JSON.stringify({
+          state: 'success',
+          id: payment_id,
+          type: 'payment',
+        }),
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        method: 'POST',
+      });
+
+      if (!response.ok) {
+        throw new Error(
+          `Payment registration has failed with status ${response.status} - ${response.statusText}`,
+        );
+      }
+
+      onSuccess();
+    } catch (error) {
+      handle(error);
+      onError(PaymentErrorMessageId.ERROR_DEFAULT);
+    }
+  }, []);
+
+  return null;
+};
+
+export default DummyPayment;
```

Comment:
It would have been good to use `git mv` for those kind of move in order to keep git history on those files.

Reply:
Yes I agree